### PR TITLE
Use valid number of backslashes, depending on 1 or 2 ticks - 4.3

### PR DESCRIPTION
**Pull request by OskarStark**
Needs rebase after #11771

### CODE REVIEW

File: reference/configuration/twig.rst [hunk 1]
```diff
@@ -112,7 +112,7 @@ called to determine the default escaping applied to the template.
 base_template_class
 ~~~~~~~~~~~~~~~~~~~
 
-**type**: ``string`` **default**: ``'Twig\\Template'``
+**type**: ``string`` **default**: ``Twig\Template``
```
Comment (OskarStark) [inline reference/configuration/twig.rst]: hmm as this is a string `'....'` looks korrekt.... but

File: reference/forms/types/collection.rst [hunk 2]
```diff
@@ -324,7 +324,7 @@ type::
 entry_type
 ~~~~~~~~~~
 
-**type**: ``string`` **default**: ``Symfony\\Component\\Form\\Extension\\Core\\Type\\TextType``
+**type**: ``string`` **default**: ``Symfony\Component\Form\Extension\Core\Type\TextType``
```
Comment (OskarStark) [inline reference/forms/types/collection.rst]: ... then we should here add `'` before and after too, right? @javiereguiluz ?

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Oskar.

File: reference/forms/types/collection.rst (hunk 2 above)
Comment (javiereguiluz) [reply reference/forms/types/collection.rst]: Yes, I added the `'` in both cases because they are strings.

**Pull request merged**